services/ui-src/src/components/tables/MobileEntityRow.tsx

```diff
@@ -36,14 +43,19 @@ export const MobileEntityRow = ({
         <Td>
           <Box sx={sx.rowHeader}>
             <EntityStatusIcon entity={entity as EntityShape} />
-            <Text>{parseCustomHtml(tableHeader)}</Text>
+            <Text>{tableHeader && parseCustomHtml(tableHeader)}</Text>
```

Comment:
This fixes an error I found where the app would error with no verbiage passed in. That will never happen outside of development, but we should still handle this somewhat gracefully.